packages/docs/package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "@pixi/react-docs",
-  "version": "7.0.0-alpha.1",
```

Comment:
We also need to update the `fallback-to-canvas.mdx` page to reflect that we deleted the legacy package

Reply:
Same with the `react-spring.mdx` page